### ISSUE TITLE
Write tests

### ISSUE THREAD

**Issue by GOATS2K** [GOATS2K/overpass#16]
This should've probably been done a long time ago...

**Comment by natehalsey** [comment]
Yes! (Just created a duplicate of this). I wouldn't mind helping out with either this, or grabbing a couple of the bugs. Looking for something to do on the weekend.

**Comment by GOATS2K** [comment]
Hey, thanks for contributing.

I'm still planning a re-write of Overpass to use FastAPI and some kind of SPA framework, so I think writing tests for the current state of the application will be wasteful.

**Comment by natehalsey** [comment]
Any ETA on the FastAPI implementation? I am totally on board with holding off. 

I guess I'll work on https://github.com/GOATS2K/overpass/issues/17 , I'm pretty sure I know what approach to take.

**Comment by GOATS2K** [comment]
>Any ETA on the FastAPI implementation? I am totally on board with holding off.

None so far, I'm busy with some other things at the moment and will not have time to properly work on Overpass until May.
